README.md

```diff
@@ -56,60 +56,15 @@ scoop install python ruby go perl
 
 If you've built software that you'd like others to use, Scoop is an alternative to building an installer (e.g. MSI or InnoSetup) — you just need to zip your program and provide a JSON manifest that describes how to install it.
 
-## Requirements
-
-- Windows 7 SP1+ / Windows Server 2008+
-- [PowerShell 5](https://aka.ms/wmf5download) (or later, include [PowerShell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6)) and [.NET Framework 4.5](https://www.microsoft.com/net/download) (or later)
-- PowerShell must be enabled for your user account e.g. `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
-
 ## Installation
 
-Run the following command from your PowerShell to install scoop to its default location (`C:\Users\<user>\scoop`)
+Run the following command from a **non-admin** PowerShell to install scoop to its default location `C:\Users\<YOUR USERNAME>\scoop`.
 
 ```powershell
-Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')
-
-# or shorter
 iwr -useb get.scoop.sh | iex
```

Comment:
```suggestion
iwr get.scoop.sh | iex
```

I don't think this param is required anymore

Reply:
It's required by PowerShell 5.